Source/WebKit/UIProcess/Launcher/glib/ProcessLauncherGLib.cpp

```diff
@@ -125,10 +125,12 @@ static bool isSandboxEnabled(const ProcessLauncher::LaunchOptions& launchOptions
 
 void ProcessLauncher::launchProcess()
 {
-    IPC::SocketPair socketPair = IPC::createPlatformConnection(connectionOptions());
+    RELEASE_ASSERT(m_launchOptions.processType != ProcessLauncher::ProcessType::DBusProxy);
```

Comment:
This is breaking the build if `BUBBLEWRAP_SANDBOX` is disabled because `ProcessLauncher::ProcessType::DBusProxy` is only defined if the sandbox is enabled.

Reply:
Yes, fix here: https://github.com/WebKit/WebKit/pull/31461

Reply:
BTW, if you're working on the Android support, I wonder if it might be simpler to have a completely separate ProcessLauncher for libwpe on non-Linux? I'm not sure. It's starting to feel like a lot of conditionals, though. :)